### PR TITLE
feat: add ability for admin to change member roles

### PR DESCRIPTION
**Pull request by venarius**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This allows admins to change the roles of their team members

Screenshot 2022-08-20 151654

### CODE REVIEW

**Comment by jainpawan21** [comment]
You rock @venarius 🚀

File: apps/web/src/api/organization.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export function getCurrentOrganization() {
 export function inviteMember(email: string) {
   return api.post(`/v1/invites`, {
     email,
-    role: MemberRoleEnum.ADMIN,
+    role: MemberRoleEnum.MEMBER,
```
Comment (scopsy) [inline apps/web/src/api/organization.ts:15]: The only thing i would change is to still invite an Admin by default, for now we don't have a well established permissions and roles management in terms of what a member can or cannot do. It dissalows to delete certain templates and invite new members. During this staged I would prefer to go with a more permissive approach of ADMIN by default as for now it's more relevant for most of the cases.

When we will do a more refined Access control we can change the default behaviour, or prompt the user on what he want to invite as.

**Review by scopsy — COMMENTED** [review]
This looks and works great! Just tested it locally too 🎉
Have a small comment about the default role and we can merge this one after the fix there

File: apps/web/src/api/organization.ts (hunk 1 above)
Comment (venarius) [reply apps/web/src/api/organization.ts]: Yeah, understandable 🙂

**Review by scopsy — APPROVED** [review]

**Pull request merged**